src/Features/Core/Portable/SolutionCrawler/ISolutionCrawlerProgressReporter.cs

```diff
@@ -18,11 +19,28 @@ internal interface ISolutionCrawlerProgressReporter
         /// <summary>
         /// Raised when solution crawler progress changed
         /// 
-        /// bool indicates whether progress is started or stopped
-        /// 
         /// Notifications for this event are serialized to preserve order. 
         /// However, individual event notifications may occur on any thread.
         /// </summary>
-        event EventHandler<bool> ProgressChanged;
+        event EventHandler<ProgressData> ProgressChanged;
+    }
+
+    internal struct ProgressData
+    {
+        public Status Type { get; }
```

Comment:
Nit: Probably better to call the property also `Status`

Reply:
I get compiler error saying Status is a type.